context.go

```diff
@@ -117,15 +120,17 @@ func (c *Context) Mul(d, x, y *Decimal) (Condition, error) {
 	d.Coeff.Mul(&x.Coeff, &y.Coeff)
 	res := d.setExponent(c, 0, int64(x.Exponent), int64(y.Exponent))
 	res |= c.round(d, d)
-	return res.GoError(c.Traps)
+	return c.goError(res)
 }
 
-// Quo sets d to the quotient x/y for y != 0. c's Precision must be > 0.
+// Quo sets d to the quotient x/y for y != 0. c.Precision must be > 0. If an
+// exact division is required, use a context with high precision and verify
+// it was exact by checking the Inexact flag on the return Condition.
 func (c *Context) Quo(d, x, y *Decimal) (Condition, error) {
 	if c.Precision == 0 {
 		// 0 precision is disallowed because we compute the required number of digits
 		// during the 10**x calculation using the precision.
-		return 0, errors.New(errZeroPrecision)
+		return 0, errors.New(errZeroPrecisionStr)
 	}
 	if c.Precision > 5000 {
 		return 0, errors.New("Quo requires Precision <= 5000")
```

Comment:
When you add artificial limits like this, add a comment saying why. The answer is almost always "performance" but its good to have that written down somewhere.